### DISCUSSION TITLE
trigger 'center column' by three-finger swipe

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [YaLTeR/niri#263] [Ideas]
currently three-finger horizontal swipe seems able to trigger:

- horizontally slide to window to the left/right
- pan current window to the left/right side of the display (if this is the right wording)

while by keyboard we only have access to the first out of those two, by keyboard we can currently also center a column to the center of the screen.
it might be cool to allow centering like that using a three-finger swipe as well.

**Comment by YaLTeR** [comment]
While it shouldn't be difficult to add, it adds more snapping points that could potentially get annoying. Will need to see how it feels.

**Comment by nebulosa2007** [comment]
Btw, what a minimal required equipment or soft needs for detecting three-fingers gestures? Two-fingers gestures works only on my laptop, what else I need to do?

**Reply by YaLTeR** [reply]
Uhh, idk, that's entirely the responsibility of libinput.
